docs/use-cases/mobile-driving-license/mdl-overview/index.md

```diff
@@ -1,7 +1,7 @@
 # Mobile Driving License Solution Overview
 
 ## The Mobile Driving License Use Case
-The European Parliament and the Council reached a provisional agreement on European Commission’s initiative to modernise EU driving licence rules \[[4th Driving License Directive](https://www.europarl.europa.eu/RegData/commissions/tran/lpag/2025/05-14/TRAN_LA(2025)773299_EN.pdf)]. The new Directive on driving licences contains several key elements, including the introduction of digital EU driving licences that can be accessed on mobile phones or other digital devices, and used throughout the entire EU. It will be issued into the EU Digital Identity Wallet. After a brief transitionary period, digital driving licences will be issued by default in all Member States. The Commission is supporting Member States to implement EU mobile driving licences, one of such support measures is to provide an implementation toolbox to facilitate implementation of mDL
+The mDL is regulated by the \[[Directive (EU) 2025/2205 (4th Driving License Directive)](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=OJ:L_202502205)]. The new Directive on driving licences contains several key elements, including the introduction of digital EU driving licences that can be accessed on mobile phones or other digital devices, and used throughout the entire EU. It will be issued into the EU Digital Identity Wallet. After a brief transitionary period, digital driving licences will be issued by default in all Member States. The Commission is supporting Member States to implement EU mobile driving licences, one of such support measures is to provide an implementation toolbox to facilitate implementation of mDL.
```

Comment:
Is there a reason for the outer `[[...]]`? 

Markdown links work without this additional layer:

```
[Directive (EU) 2025/2205 …](https://eur-lex.europa.eu/…)
```
In technical document we do not have any reason to visually enclose these links in brackets. Additionally, they introduce noise, reduce readability and break the visual flow